creator-node/src/fileManager.js

```diff
@@ -119,19 +119,16 @@ async function saveFileForMultihash (req, multihash, expectedStoragePath) {
   // If multihash already pinned by local INode, cat file from local ipfs node
   req.logger.info(`checking if ${multihash} already pinned by local ipfs node`)
   try {
-    const pinset = await ipfs.pin.ls(multihash)
-    if (pinset.includes(multihash)) {
-      req.logger.info(`File for ${multihash} already pinned by local ipfs node`)
-      fileBuffer = await ipfs.cat(multihash)
-      req.logger.info(`Retrieved file for ${multihash} from local ipfs node`)
-    }
+    fileBuffer = await ipfs.cat(multihash)
```

Comment:
bc we call export immediately before this, ALL files should be available through cat on the node performing sync

Reply:
worth thinking through this scenario however

Reply:
yeah this should be available through the other primary. however, we could also have a race here and maybe have a fallback to try the creator node gateway from the primary?